typescript/cli/src/utils/env.ts

```diff
@@ -0,0 +1,17 @@
+import z from 'zod';
+
+const envScheme = z.object({
+  HYP_KEY: z.string().optional(),
+});
+
+const parsedEnv = envScheme.safeParse(process.env);
+
+if (!parsedEnv.success)
+  throw new Error(
+    `Failed to parse environment variables: ${JSON.stringify(
+      parsedEnv.error.format(),
+      null,
+    )}`,
+  );
```

Comment:
Either way is fine but I'd be more inclined to fallback to an empty ENV object instead of throwing here. Otherwise this is equivalent to using `parse()`. The idea is to avoid a fatal error based on weird content in a user's ENV vars

Reply:
Ah yep that's fair. FWIW I've got plenty of non-HL env vars that didn't pose an issue when running locally. Regardless will simply have this return `{}` if !success